crates/core/src/octree/build/stages/flag_nodes.rs

```diff
@@ -72,19 +72,13 @@ mod tests {
     fn flag_nodes_works() {
         let (_glfw, _window) = test_utils::init_opengl_context();
 
-        // To go from the crate root to the workspace root
-        let mut path = PathBuf::from(env::current_dir().unwrap());
-        path.pop();
-        path.pop();
-        env::set_current_dir(path).unwrap();
-
```

Comment:
A hack we could do is save the cwd, do this at the start and then go back to the previous cwd at the end of the test. I'll try to do that. Created https://github.com/franciscoaguirre/voxel-cone-tracing/issues/99 for it